src/util/config.rs

```diff
@@ -452,6 +454,11 @@ impl Serialize for ReadableDuration {
     {
         let mut dur = util::time::duration_to_ms(self.0);
         let mut buffer = String::new();
+        // For backward compatibility, do not render config that old version cannot parse.
+        // if dur >= DAY {
```

Comment:
Remove the commented out implementation.

Reply:
It is a cherry pick. If we just delete them, then it is hard to know what we change and why it is not consistent with the master branch.